### DISCUSSION TITLE
Failed to connect to the codespace

### DISCUSSION THREAD

**Discussion by aaronrubinstein** [community/community#43016] [Codespaces] [answered]
### Select Topic Area

Bug

### Body

I just started using codespaces the other day and it's been fantastic, however all of a sudden when I try to open a codespace it briefly tries to connect and then returns a failure message:
![image](https://user-images.githubusercontent.com/40306940/210159113-5c726065-ba43-4b4a-9633-345318da100c.png)

Here's what I'm seeing in the console:

![image](https://user-images.githubusercontent.com/40306940/210159121-b0670062-d0d5-485b-bd74-5b05cc386655.png)

After I try to connect and get this message, I see that the codespace becomes active on on github.com/codespaces, however I still can't connect to it. 

Things I've tried:
- Stopping the codespace and reopening it
- Creating a new codespace: I get the same failure message or sometimes is just hangs on the "Connecting..." message
- Using a different browser: same issue in Chrome and Edge
- Using a different machine: I tried a different laptop but still have the same issue

**Comment by ghost** [comment]
Same for me, it was fine then the same thing happened

**Comment by Goody1981** [comment]
I have the same issue... was fine yesterday and now cannot connect. Repeatedly got the "Try again" message @aaronrubinstein mentions above.

I tried deleting browser history/cache, and I no longer got the "Try again" message, instead it just stays forever on the "Setting up your codespace" message (screenshot below)

![image](https://user-images.githubusercontent.com/119047651/210160339-56e1e7e2-b628-4b56-994a-a50028bf33df.png)

**Comment by ghost** [comment]
@Goody1981 same problem with me it stays on "connecting''  and "setting up your codespace..."

this is the status page for this incident from 12/31/22
![image](https://user-images.githubusercontent.com/115805897/210160825-6e5e339a-bb8a-4861-a1da-e0f134ad71ea.png)

**Reply by d4v3-h4ck3r** [reply]
Mine stopped working now... did they fix yours?

**Comment by ghost** [comment]
btw happy new year everyone :)

**Comment by Goody1981** [comment]
@l3ViaThA09 happy new year :)

Awesome, didn't realise this was an issue they already knew about, cheers for pointing that out. Not working for me yet so perhaps the fix is still coming...

**Comment by jfullstackdev** [comment]
same, I'm waiting for it to be okay though there is no report in GitHub Status 

![image](https://user-images.githubusercontent.com/47092464/210161072-f58cb4ce-ed11-4f40-a777-6916f68585a6.png)

**Comment by SharpIceX** [comment]
Maybe it's just a problem with the vscode service.
I use `gh cs ssh`
can still connect
![图片](https://user-images.githubusercontent.com/61524717/210161230-0a5b1e9c-1116-4725-9b2d-89dd2a4ab152.png)

**Comment by nirajulislam** [comment]
Mine too also is not working

**Comment by RexRowan** [comment]
Same problem, connecting forever

**Comment by Amirkia92** [comment]
same here!

**Comment by Yarden-zamir** [comment]
Having the same issue, still not fixed. Working fine on desktop app but not on the web.
![Screen Shot 2023-01-01 at 13 10 33](https://user-images.githubusercontent.com/8178413/210168569-ee3c7509-a0be-4a20-80f0-9f9473897d4a.png)
![Screen Shot 2023-01-01 at 13 10 44](https://user-images.githubusercontent.com/8178413/210168580-851aa3dd-5d27-4441-b2c4-4ca09e0d2972.png)
![Screen Shot 2023-01-01 at 13 11 04](https://user-images.githubusercontent.com/8178413/210168597-faa8ac5b-01ed-40c4-adf5-70e30f7758f0.png)

**Comment by ghost** [comment]
Same issue here! I have an assignment to submit before 12 am! Please fix this please! All my file are there :(
![Screenshot_20230101_155734_Chrome](https://user-images.githubusercontent.com/119036040/210169314-a6e8feef-6244-444d-a36f-b61f4aa26e11.jpg)

**Reply by ghost** [reply]
Can someone tell me how to atleast get my files from my codespace? I'm kinda new here

**Reply by jaeho0718** [reply]
Use vscode app.

**Reply by Yarden-zamir** [reply]
The app seems to work, here https://code.visualstudio.com/download

**Reply by ghost** [reply]
Is there an android version of the app?

**Reply by Yarden-zamir** [reply]
Not officailly, it might be possible (on android) but will be painful, you don't have access to any computer?

**Reply by SharpIceX** [reply]
> 该应用程序有安卓版本吗？

Android maybe this one
https://vscode.dev/

**Reply by ghost** [reply]
> Not officailly, it might be possible (on android) but will be painful, you don't have access to any computer?

I do but it isn't free at the moment and I really need to get the files from my codespace

**Comment by Yarden-zamir** [comment]
I see the option to export all current changes into a branch, seems to work for me. @FadingDarkness-8449 does this work for you?
![Screen Shot 2023-01-01 at 14 34 23](https://user-images.githubusercontent.com/8178413/210170712-271f0bc2-cbe5-4ce1-8e91-cb5409232083.png)

**Comment by yxshv** [comment]
same for me, doesn't connect

have been trying since morning to check if they fixed it.

**Comment by mlozanoqf** [comment]
Same problem, connecting forever.

**Comment by gigamonkey** [comment]
Just chiming in to say this is still happening as of 2023-01-01T16:03Z. Can't connect to any codespaces in the browser. Connecting in VS Studio Code does work.

**Comment by ghost** [comment]
@github-staff

**Comment by ghost** [comment]
Everyone 
![image](https://user-images.githubusercontent.com/115805897/210181168-2349a8b1-7eab-44d2-939c-9fea76a224e9.png)
Update from github status

**Comment by ghost** [comment, marked as answer]
More Updates


![image](https://user-images.githubusercontent.com/115805897/210181211-f38c6f19-91e5-40ae-8d04-29689d6f33fc.png)

**Comment by yxshv** [comment]
its fixed. i can connect now

**Reply by Mahad-Haqqi** [reply]
how did you solve the issue?

**Reply by Yarden-zamir** [reply]
The issue seems to be solved on github side, it should work for you as well. If not try a different region maybe

**Comment by jfullstackdev** [comment]
yup, it's fixed by now

**Comment by aaronrubinstein** [comment]
Confirmed it's working for me now too 👍

**Reply by Face248444** [reply]
how to fix this problem? please help me.

**Comment by Face248444** [comment]
how to fix this problem? please help me.
[38e9687c-7706-41c5-b783-897339c14905.webm](https://user-images.githubusercontent.com/108752331/211147575-1fcf5031-0070-46d4-a786-ad8ff80e930c.webm)

**Reply by Face248444** [reply]
please help me
